src/box/lua/init.c

```diff
@@ -227,6 +228,7 @@ static const char *lua_sources[] = {
 	"box/upgrade", NULL, upgrade_lua,
 	"box/net_box", "net.box", net_box_lua,
 	"box/net_replicaset", "internal.net.replicaset", net_replicaset_lua,
+	"box/net_cluster", "experimental.net.cluster", net_cluster_lua,
```

Comment:
I think that a documentation request is desirable for experimental modules. This way engages users into testing and leaving a feedback.

I also think that an item in the changelog/release notes is good too as a little advertisement.